include/dlaf/auxiliary/norm/mc.h

```diff
@@ -15,14 +15,39 @@
 #include <dlaf/auxiliary/norm/api.h>
 #include <dlaf/common/range2d.h>
 #include <dlaf/common/vector.h>
+#include <dlaf/communication/kernels/reduce.h>
 #include <dlaf/communication/sync/reduce.h>
 #include <dlaf/lapack/tile.h>
 #include <dlaf/matrix/distribution.h>
+#include <dlaf/sender/transform_mpi.h>
 #include <dlaf/types.h>
 #include <dlaf/util_matrix.h>
 
 namespace dlaf::auxiliary::internal {
 
+template <class T>
+pika::execution::experimental::unique_any_sender<T> reduce_in_place(
+    pika::execution::experimental::unique_any_sender<dlaf::comm::CommunicatorPipelineExclusiveWrapper>
```

Comment:
Consider ETI for this?

Reply:
What would you suggest? I'm not sure I want to move it outside the norm, because for the moment is a super specific use-case (working with a single value). So I'd keep it there. So, about ETI, would you add the instantiation in the norm file?

Reply:
We quickly discussed about this with @msimberg offline, and since at the moment it is in the implementation header and we are not interested in using it elsewhere, this gets already instantiated by the norm ETI.